docs/DEV.md

```diff
@@ -79,3 +81,86 @@ go build && ./inlets-operator \
   --access-key="ACCESS_KEY"
 ```
 
+## Run the go binary with Azure and Rancher Desktop
+
+Go to [Rancher Desktop](https://rancherdesktop.io/) and download and install Rancher Desktop.
+
+Select your kubernetes version and take care to choose the `rancher-desktop` as kubernetes context
+
+![img.png](images/rancher-desktop.png)
+
+Sign Up at [Azure Portal](https://azure.microsoft.com/en-us/free/search/).
```

Comment:
We already have this at: https://docs.inlets.dev/#/tools/inlets-operator?id=inlets-operator-reference-documentation